projects/aca-shared/src/lib/services/app.service.ts

```diff
@@ -166,9 +166,11 @@ export class AppService implements OnDestroy {
       if (isReady) {
         this.loadRepositoryStatus();
         this.loadUserProfile();
-        setTimeout(() => {
-          this.openMobileAppDialog();
-        });
+        if (document.readyState === 'complete') {
+          setTimeout(() => {
```

Comment:
We should have tests for bug fixes to prevent the same problem in the future. Can we add some tests?